src/main/webapp/app/course/learning-paths/components/learning-path-lecture-unit/learning-path-lecture-unit.component.html

```diff
@@ -6,21 +6,30 @@
     </div>
 }
 @if (lectureUnit(); as lectureUnit) {
-    @switch (lectureUnit.type) {
-        @case (LectureUnitType.VIDEO) {
-            <jhi-video-unit [lectureUnit]="lectureUnit" (onCompletion)="setLearningObjectCompletion($event)" />
-        }
-        @case (LectureUnitType.TEXT) {
-            <jhi-text-unit [lectureUnit]="lectureUnit" (onCompletion)="setLearningObjectCompletion($event)" />
-        }
-        @case (LectureUnitType.ATTACHMENT) {
-            <jhi-attachment-unit [lectureUnit]="lectureUnit" (onCompletion)="setLearningObjectCompletion($event)" />
-        }
-        @case (LectureUnitType.EXERCISE) {
-            <jhi-exercise-unit [exerciseUnit]="lectureUnit" />
-        }
-        @case (LectureUnitType.ONLINE) {
-            <jhi-online-unit [lectureUnit]="lectureUnit" (onCompletion)="setLearningObjectCompletion($event)" />
+    <div class="row m-0">
+        <div class="col p-0">
+            @switch (lectureUnit.type) {
+                @case (LectureUnitType.VIDEO) {
+                    <jhi-video-unit [lectureUnit]="lectureUnit" (onCompletion)="setLearningObjectCompletion($event)" />
+                }
+                @case (LectureUnitType.TEXT) {
+                    <jhi-text-unit [lectureUnit]="lectureUnit" (onCompletion)="setLearningObjectCompletion($event)" />
+                }
+                @case (LectureUnitType.ATTACHMENT) {
+                    <jhi-attachment-unit [lectureUnit]="lectureUnit" (onCompletion)="setLearningObjectCompletion($event)" />
+                }
+                @case (LectureUnitType.EXERCISE) {
+                    <jhi-exercise-unit [exerciseUnit]="lectureUnit" />
+                }
+                @case (LectureUnitType.ONLINE) {
+                    <jhi-online-unit [lectureUnit]="lectureUnit" (onCompletion)="setLearningObjectCompletion($event)" />
+                }
+            }
+        </div>
+        @if (isCommunicationEnabled() || isMessagingEnabled()) {
```

Comment:
Then, you can simplify this condition:

```suggestion
        @if (isCommunicationEnabled()) {
```